command/job_dispatch_test.go

```diff
@@ -1,6 +1,7 @@
 package command
 
 import (
+	structs2 "github.com/hashicorp/nomad/nomad/structs"
```

Comment:
Not sure if you need to alias the package name.

```suggestion
	"github.com/hashicorp/nomad/nomad/structs"
```